### PR TITLE
Update README & Contributing files

### PR DESCRIPTION
**Pull request by diabawada**
Update README & Contributing files. Version to v0.0.6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,6 @@ This guide describes how to use our various Authentication Methods to fetch [Sta
 | **gcp-audience**                  | No       | `string`  | GCP audience to use in signed JWT (relevant only for access-type=`gcp`).                                                     |
 | **gateway-url**                   | No       | `string`  | Gateway URL for the K8s authenticated (relevant only for access-type=`k8s`/`oauth2`).                                          |
 | **k8s-auth-config-name**          | No       | `string`  | The K8s Auth config name (relevant only for access-type=`k8s`).                                                              |
-| **k8s-service-account-token**     | No       | `string`  | The K8s service account token (relevant only for access-type=`k8s`).                                                         |
```
Comment (amirmao) [inline README.md:58]: 👍

File: Contributing.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ We warmly welcome contributions to the Akeyless GitHub Actions Plugin! Whether i
 Fork this repository on GitHub and clone your fork locally.
 
 ```
-git clone [your-fork-url]
+git clone https://github.com/akeyless-community/akeyless-github-action.git
 cd [repository-name]
```
Comment (amirmao) [inline Contributing.md]: WDYT changing the repository-name to `akeyless-github-action` (or the default one that is been created when doing the clone like you wrote)?

**Review by amirmao — APPROVED** [review]
LGTM - one suggestion

File: Contributing.md (hunk 2 above)
Comment (diabawada) [reply Contributing.md]: @amirmao changing to akeyless-github-action like ex: `git clone akeyless-github-action ? `

Comment (amirmao) [reply Contributing.md]: `cd akeyless-github-action`

**Review by ak-nir — APPROVED** [review]

**Pull request merged**